cmake/FindONNX.cmake

```diff
@@ -0,0 +1,31 @@
+ocv_clear_vars(HAVE_ONNX)
+
+set(ORT_INSTALL_DIR "" CACHE PATH "ONNX Runtime install directory")
+
+if(ORT_INSTALL_DIR)
+  find_library(ORT_LIB onnxruntime
+    ${ORT_INSTALL_DIR}/lib
+    CMAKE_FIND_ROOT_PATH_BOTH)
```

Comment:
> CMAKE_FIND_ROOT_PATH_BOTH

Do we really need that?

Reply:
Have no idea, tbh. Probably it works even without it.